public/app/features/dashboard-scene/scene/CustomTimeRangeCompare.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { SceneTimeRangeCompare, SceneComponentProps } from '@grafana/scenes';
+
+export class CustomTimeRangeCompare extends SceneTimeRangeCompare {
```

Comment:
Don't understand what is this for?

Reply:
I was having trouble getting it to render without a wrapper. I'l see if I can fix it.

Reply:
@dprokop I removed some of the custom rendering. Now the wrapper only handles panel option changes.